### PR TITLE
#168049994 Setup docker for backend application

### PR DESCRIPTION
**Pull request by Jordybastien**
#### What does this PR do?
It contains a docker file that would create an image to run the application
#### Description of Task to be completed?

- Tell it Image to build from

- Create App directory

- Copy package.json

- Install all dependencies

- Bundle app source

- Map port to the Docker daemon

- Run the start script

#### How should this be manually tested?
By building the image
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[168049994](https://www.pivotaltracker.com/story/show/168049994)
#### What are the relevant Github Issues?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by amilykassim — COMMENTED** [review]
@Jordybastien can you add an empty line at the end of the docker file for the sake of standards, 
besides that, it looks good to me.

**Comment by Jordybastien** [comment]
> @Jordybastien can you add an empty line at the end of the docker file for the sake of standards,
> besides that, it looks good to me.

On it

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+RUN npm install
+
+# Bundle app source
+COPY . .
+
+# Map port to the Docker daemon
+EXPOSE 3000
+
+# Run the start script
+CMD [ "npm", "dev" ]
```
Comment (mucyomiller) [inline Dockerfile:20]: you are running 
```shell 
npm dev 
``` 
inside docker container but right now we don't have it in package.json scripts, either put it there or this PR has to wait for @amilykassim's work to get marge. thanks

Comment (Jordybastien) [reply Dockerfile:20]: That was the idea, waiting for @amilykassim PR, but let me just add it there too, whatever PR that get merged first we'll know what to do

**Review by gik-hub — APPROVED** [review]
Okay

**Review by mucyomiller — APPROVED** [review]
LGTM

**Review by gik-hub — APPROVED** [review]
Okay

**Review by amilykassim — APPROVED** [review]
LGTM

**Review by Paulvitalis200 — APPROVED** [review]

**Pull request merged**